packages/flutter_tools/templates/app_shared/web/index.html.tmpl

```diff
@@ -30,7 +30,7 @@
   <link rel="icon" type="image/png" href="favicon.png"/>
 
   <title>{{projectName}}</title>
-  <link rel="manifest" href="manifest.json">
+  <link rel="manifest" href="manifest.json" crossorigin="use-credentials">
```

Comment:
Wouldn't it be simpler to just comment above the `link rel="manifest"` something like:

```html
<!-- If your app is protected behind simple http auth, you need to set crossorigin="use-credentials" in the following tag. -->
```

That way we don't flip the default for a use case that is not that common?

Reply:
I don't have an opinion, would you like me to change the PR?

Reply:
according the @ditman , I think yes